### ISSUE TITLE
Allow More Editing

### ISSUE THREAD

**Issue by ericthelin** [imrahil/OctoPrint-PrintHistory#29]
I would like to have the ability to edit the data for completed prints to manually change success/fail or to manually set the filament used when it is blank because octoprint didn't calculate it.

**Comment by CptanPanic** [comment]
Yes I agree, would be nice to be able to change a print to failed if it didn't actually come out correct.

**Comment by gheydon** [comment]
Yes being able to fail a print will mean that your print failed/succeed ratio will be much more accurate.
